astropy/coordinates/angles/utils.py

```diff
@@ -4,14 +4,7 @@
 used internally in astropy.coordinates.angles, and of possible.
 """
 
-__all__ = [
-    "angular_separation",
-    "position_angle",
-    "offset_by",
-    "golden_spiral_grid",
-    "uniform_spherical_random_surface",
-    "uniform_spherical_random_volume",
-]
+__all__ = []
```

Comment:
Can we leave this as is until we have resolved the APE?

Reply:
Leaving it is introducing more public API. I was trying to keep the module name not underscored without explicitly introducing new public import paths.

Reply:
But the current mode is that `__all__` defines things that are meant to be importable in `__init__.py`, using an `from xxx import *`. This is helpful for developers and does not affect users (who should never get here...). Indeed, in principle, minimal change would be to keep the `from xxx import *` in `__init__.py` as well (certainly, for `units` that seems a more sensible approach...), but I am fine with those changes. But I really dislike `__all__ = []`... (I think I'd rather have the private names...). Anyway, let's leave this for after APE 22.

Reply:
I also much prefer having informative ``__all__``!  With underscored modules we get to have both full ``__all__``, use star imports, and clearly delineate API even at the level of tab-autocomplete.

Reply:
✅ done.